.github/workflows/master_rancher_ui_experimental_workflow.yml

```diff
@@ -83,8 +56,10 @@ jobs:
           HELM_VERSION: 3.7.0
           K3S_VERSION: v1.24.6+k3s1
           EXT_REG: ${{ inputs.ext_reg }}
-          EXT_REG_USER: ${{ secrets.ext_reg_user }}
-          EXT_REG_PASSWORD: ${{ secrets.ext_reg_password }}
+          # EXT_REG_USER: ${{ secrets.ext_reg_user }}
+          # EXT_REG_PASSWORD: ${{ secrets.ext_reg_password }}
+          EXT_REG_USER: secrets.EPINIO_DOCKER_USER
```

Comment:
EXT_REG_* have wrong syntax and are useless here, please remove both

Reply:
Note, I believe `EXT_REG_USER` is needed since it fails when removed https://github.com/epinio/epinio-end-to-end-tests/actions/runs/3894717213/jobs/6649087271#step:5:125
`EXT_REG` can be removed. I will bring back `EXT_REG_USER`